docs/guides/misc/benchmark.md

```diff
@@ -21,21 +21,19 @@ They are extracted into the list file as they are recorded in the database. Ther
 
 We suggest disabling both logging and the long-term database during the benchmark run as both the log file and the database would otherwise unnecessarily grow, several hundred megabytes may be possible. Not only would your statistics be distorted by the artificial mass querying, but the benchmark could also be negatively affected by the writing speed of your SD card.
 
-Logging can be disabled using `sudo pihole logging off`.
+Logging can be disabled using `pihole-FTL --config dns.queryLogging false`.
 
-The long-term database can be disabled by setting
-
-```
-DBFILE=
-```
-
-in `/etc/pihole/pihole-FTL.conf` and running `sudo pihole restartdns` (see also [here](../../ftldns/configfile.md/#dbfile)).
+The long-term database can be disabled running `pihole-FTL --config files.database ""`
```

Comment:
I did just realize that this disabled everything from the database, network table, for instance. We should test how the API behaves, I guess we may see many error 500s which seems undesirable... I don't think we have tested this configuration on conjunction with the web interface (at least, not me).

Reply:
Setting the maximum number of days to retain in the database to zero should have the save effect (sorting no queries) without such adverse effects.

Reply:
Suggestion 1:
```suggestion
The same effect can be achieved if you set the *maximum number of days to retain in the database* to zero, running `pihole-FTL --config database.maxDBdays 0`.
```

Suggestion 2:
```suggestion
The same effect can be achieved if you set `database.maxDBdays` to zero, running `pihole-FTL --config database.maxDBdays 0`. This will change the maximum number of days the database will keep to zero, resulting in no queries being stored.
```